moxy/org.eclipse.persistence.moxy/src/test/java/org/eclipse/persistence/testing/moxy/unit/jaxb/compiler/SchemaGeneratorTestCase.java

```diff
@@ -65,4 +77,51 @@ public void addTransformerToSchema(final @Mocked Helper helper, final @Mocked Pr
 
         Deencapsulation.invoke(schemaGenerator, "addTransformerToSchema", property, typeInfo, typeDefParticle, complexType, schema);
     }
+   
+    @Test
+    public void test(final @Mocked Helper helper) {
```

Comment:
Could You please change this method name into something more meaningful. This name is too generic.